frigidaire/__init__.py

```diff
@@ -38,6 +38,34 @@ class Destination(str, Enum):
     DEHUMIDIFIER = "DH"
 
 
+def _get_destination_from_model_name(model_name: str) -> Destination:
+    """
+    Maps known model names to their corresponding destination types.
+    Falls back to direct enum lookup for backward compatibility.
+    
+    :param model_name: The model name from the appliance data
+    :return: The appropriate Destination enum value
+    :raises ValueError: If the model name is not recognized
+    """
+    # Known model name mappings
+    model_mappings = {
+        "Husky": Destination.DEHUMIDIFIER,
+        # Add more model mappings here as they are discovered
+    }
+    
+    # Check if it's a known model name first
+    if model_name in model_mappings:
+        return model_mappings[model_name]
+    
+    # Fall back to direct enum lookup for backward compatibility
+    try:
+        return Destination(model_name)
+    except ValueError:
+        raise ValueError(f"'{model_name}' is not a recognized model name or destination type. "
```

Comment:
Can we update the error message here to indicate that users should add additional mappings for their device with a link to the place to add it? That'd likely turn some would-be issues into PRs